dist/swagger.yaml

```diff
@@ -4,6 +4,11 @@ schemes: [https, http]
 consumes: [application/json]
 produces: [application/json]
 basePath: /api-v2
+securityDefinitions:
+  TokenSecurity: {in: header, type: apiKey, name: dpn_token, description: DPN Authorization
+      Token}
+security:
+- TokenSecurity: []
```

Comment:
This looks different to what I have and the name of the `security` is important.  So, what I have is:
```
    "security": [
      { "dpn_token": [] }
    ],
```
Note how it refers to the `dpn_token` name defined in the `securityDefinitions`.  That's important for the swagger-ui javascipt to work as expected.  I have no idea how it could work in the swagger-editor without the custom javascript.  We can slack chat about the details to clarify.